Cargo.toml

```diff
@@ -15,11 +15,12 @@ rustc_version = "0.2"
 rustc-hex = { version = "1.0", optional = true }
 heapsize = { version = "0.4", optional = true }
 byteorder = { version = "1", default-features = false }
+crunchy = "0.1.5"
```

Comment:
Ok, leave it as it is
I will update myself if something goes wrong in no-std binaries